avro4s/src/main/scala/com/ovoenergy/kafka/serialization/avro4s/Avro4sSerialization.scala

```diff
@@ -265,10 +281,8 @@ private[avro4s] trait Avro4sSerialization {
                                                              isKey: Boolean): KafkaDeserializer[T] =
     avroJsonSchemaIdDeserializer(SchemaRegistryClientSettings(schemaRegistryEndpoint), isKey)
 
-  def avroJsonSchemaIdDeserializer[T: FromRecord: SchemaFor](
-    schemaRegistryClientSettings: SchemaRegistryClientSettings,
-    isKey: Boolean
-  ): KafkaDeserializer[T] = {
+  def avroJsonSchemaIdDeserializer[T: FromRecord: SchemaFor](schemaRegistryClientSettings: SchemaRegistryClientSettings,
```

Comment:
Is it scala-style formatting? I preferred the previous one.

Reply:
I prefer the old one as well, Have to play around with scalafmt to make it work